installer/install.iss

```diff
@@ -1587,10 +1587,14 @@ begin
     end;
 
     AppDir:=ExpandConstant('{app}');
+    if Builtin then
```

Comment:
Don't we need to switch the then and else parts? Because if `Builtin` is set to true, we need to target the wrapper which is found under `AppDir+'\bin\git.exe'`

Reply:
No, the builtins are the Git builtins in `mingw64\libexec\git-core`. They need to be hard-linked to the real `mingw64\bin\git.exe`, really.

Git LFS, in contrast, is not a builtin, and needs to be hard-linked to the Git wrapper. I should make it more obvious that the `bin\git.exe` is the Git wrapper, really... and I guess I make that more obvious by using `cmd\git.exe` instead...

Reply:
Uff I'm so confused about what needs to be hard-linked to the wrapper and what to real git...

This would then mean that in the Git LFS case we have to pass `False` to HardlinkOrCopyGit on Line [1845](https://github.com/git-for-windows/build-extra/pull/174/commits/40143f2ad4d311bc6caccba6ecffd623831cb197#diff-9db0b81cc5d2dc9d49e4e8634ac624b6R1845) whereas we currently pass `True`

Reply:
Oy. You know what? I should totally learn how to read my own patches. Sorry about that.